crypto/src/asset.rs

```diff
@@ -28,8 +34,62 @@ use crate::Fq;
 #[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord)]
 pub struct Id(pub Fq);
 
+/// Validator-specific unique assets, used for identifying staking tokens
+/// associated with a particular validator.
+pub struct ValidatorAsset {
```

Comment:
Could we call it a `ValidatorDenom`, so the name is more clear that it's a validator-specific analog of the `Denom` type?

Reply:
comment obsoleted by https://github.com/penumbra-zone/penumbra/issues/178#issuecomment-992214707